utilities/xmlrpcpp/src/XmlRpcUtil.cpp

```diff
@@ -126,10 +130,13 @@ XmlRpcUtil::parseTag(const char* tag, std::string const& xml, int* offset)
 bool 
 XmlRpcUtil::findTag(const char* tag, std::string const& xml, int* offset)
 {
+  if (xml.length() > __INT_MAX__) return false;
   if (*offset >= int(xml.length())) return false;
   size_t istart = xml.find(tag, *offset);
   if (istart == std::string::npos)
     return false;
+  if ((istart + strlen(tag)) > __INT_MAX__)
+    return false;
 
   *offset = int(istart + strlen(tag));
```

Comment:
I suggest calculating this once in a size_t and comparing to `__INT_MAX__` then casting to int rather than calculating it twice.

Reply:
Addressed in 2404bd7fb0c2fb60c1950bf7101b200723660a0a